addons/stock/wizard/stock_change_product_qty.py

```diff
@@ -86,9 +86,8 @@ def change_product_qty(self):
                 inventory_filter = 'none'
             inventory = Inventory.create({
                 'name': _('INV: %s') % tools.ustr(wizard.product_id.display_name),
-                'filter': inventory_filter,
-                'product_id': wizard.product_id.id,
-                'location_id': wizard.location_id.id,
+                'product_ids': [(4, wizard.product_id.id)],
```

Comment:
@svs-odoo shoudln't this wizard be erased completely?

Reply:
Yes it must be, I forgot it. I delete it, thanks